opentelemetry-tracing/opentelemetry-dotnet-manual/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenTelemetry .NET SDK Example
```

Comment:
How about creating an additional directory like `aspnetcore-and-mongodb`?

Then we could have two README files where in one we could describe to approach of doing manual instrumentation using the SDK and the second would describe the example application. This would look more similar to other languages https://github.com/signalfx/tracing-examples/tree/main/opentelemetry-tracing.

Reply:
I didn't notice that structure... fixing it...

Reply:
Moved via [b45b791](https://github.com/signalfx/tracing-examples/pull/112/commits/b45b791f851f58585970654b4a9388eb0c20a464). Keeping this unresolved until I have a chance to update the README files.